README.md

```diff
@@ -1,154 +1,139 @@
-# Nerves.Runtime
+# Nerves Runtime
```

Comment:
My thinking here is that "Nerves Runtime" is one of the "products" under the umbrella of the Nerves Project. We don't call it "Nerves-dot-runtime" or "Nerves underscore runtime" when we're talking about it; just "Nerves Runtime" (with a capital "R" because it's not the general runtime of Nerves, but the specific library that we call "Nerves Runtime.") I think in the Hex docs, it makes total sense to document `Nerves.Runtime` as a module, but my opinion is that the project document is about the "product" and not just the Elixir module that embodies it (e.g. there are often several modules included in a repository).